pxtlib/cpp.ts

```diff
@@ -836,8 +838,11 @@ namespace pxt.cpp {
 
         res.generatedFiles[sourcePath + "pointers.cpp"] = includesInc + protos.finish() + abiInc + pointersInc + "\nPXT_SHIMS_END\n"
         res.generatedFiles[sourcePath + "pxtconfig.h"] = pxtConfig
-        if (isYotta)
+        pxt.debug(`pxtconfig.h: ${res.generatedFiles[sourcePath + "pxtconfig.h"]}`)
+        if (isYotta) {
             res.generatedFiles["/config.json"] = JSON.stringify(configJson, null, 4) + "\n"
+            pxt.debug(`yotta config.json: ${res.generatedFiles["/config.json"]}`)
+        }
         res.generatedFiles[sourcePath + "main.cpp"] = `
 #include "pxt.h"
 #ifdef PXT_MAIN
```

Comment:
I'm not sure this is actually used: https://github.com/Microsoft/pxt-common-packages/blob/master/libs/base/pxtbase.h#L676

Reply:
we are not releasing the fiber in there?

Reply:
yes, we are https://github.com/Microsoft/pxt-common-packages/blob/master/libs/base/pxt.cpp#L535